docs/frontend/introduction-to-less.md

```diff
@@ -0,0 +1,112 @@
+# Indtroduction to LESS
+This document describes behavior of CSS pre-processor [LESS](http://lesscss.org/).
+
+## Import files
+In Shopsys Framework we implement `LESS` by dividing styles into many `LESS components`. Each component has its own file. Filename and component name are the same. We use `@import` command for joining all components contents into one compiled `CSS` file.
+
+Here we can see syntax of `@import` command.
+```less
+@import 'path/to/directory/component-filename.less';
+```
+
+### Usage of @import command
+Best way how to import all related files is to create one file, for example named as `main.less`. This file will contain only `@import` commands. Keep in mind where you place this file. Imported path depends on where this file is placed.
+
+#### Unexpected behavior
+There is one thing which you should keep in mind. When you try to import file which does not exist in given file path, compiler will try to find missing file in root directories of files, where is used `@import`.
+
+Let us show this at example. Assuming you have folder structure and files as it is shown below.
+```
+│── root-main.less
+│── some-component.less
+└─── B
+    └── b-main.less
+```
+
+```less
+/** B/b-main.less */
+
+@import "some-component.less";
+```
+
+```less
+/* some-component.less */
+
+.some-component {
+    color: red;
+}
+```
+
+```less
+/* root-main.less */
+
+@import "B/b-main.less";
+```
+
+Result CSS of this example will be.
+```less
+.some-component {
+    color: red;
+}
+```
+As an explanation of this behavior, given in example above, is that compiler is trying to find a file `some-component.less` firstly in folder where is placed `b-main.less`, then in directory of `root-main.less`. When it could not find required file in any directory, then it will thrown an error during compiling.
```

Comment:
There are a few missing articles ("a/an/the") in your document, I suggest using https://app.grammarly.com :slightly_smiling_face:

Reply:
Thanks for the tip. :)